src/main/java/org/openrewrite/java/template/RefasterTemplateProcessor.java

```diff
@@ -94,7 +94,8 @@ public class RefasterTemplateProcessor extends AbstractProcessor {
 
     static Set<String> DO_AFTER_VISIT = Stream.of(
             "new org.openrewrite.java.ShortenFullyQualifiedTypeReferences().getVisitor()",
-            "new org.openrewrite.java.cleanup.UnnecessaryParenthesesVisitor()"
+            "new org.openrewrite.java.cleanup.UnnecessaryParenthesesVisitor()",
+            "new org.openrewrite.staticanalysis.SimplifyBooleanExpression().getVisitor()"
```

Comment:
This visitor also still exists in `rewrite-java` as `org.openrewrite.java.cleanup.SimplifyBooleanExpressionVisitor`. I think it we could avoid requiring a dependency on `rewrite-static-analysis`.

Reply:
Thanks for pointing that out; seems we have _two_ full implementations, one in rewrite, and another in rewrite-static-analysis, likely because the second one can access a Kotlin marker that the first one cant. We should likely merge those two at some point.